subsys/usb/class/cdc_acm.c

```diff
@@ -118,15 +118,15 @@ struct usb_cdc_acm_config {
 #define INITIALIZER_IF_HDR						\
```

Comment:
I think we also can move initializers to the common header.

Reply:
For some yes, but there exist a group of class specific ones which better stay here. We can consider this change later and decide they best solution.